tests/data/test_study/extract_configs/simple_tsv_example1.py

```diff
@@ -28,6 +28,11 @@ def observed_yes_no(x):
         },
         out_col=CONCEPT.PARTICIPANT.ID
     ),
+    value_map(
+        in_col="participant",
+        m=lambda x: 'PT_3QMCQXHZ' if x == 'PID001' else None,
+        out_col=CONCEPT.PARTICIPANT.TARGET_SERVICE_ID
+    ),
```

Comment:
lol that's one way to do it. I expect someone setting kfids to make spreadsheet with columns for pid and kfid.

Reply:
Yea this was just for testing